lib/document/paged-rendering.js

```diff
@@ -5,7 +5,33 @@ class PagedReadyHandler extends Paged.Handler {
   afterRendered (pages) {
     window.AsciidoctorPDF.status = 'ready'
   }
+
+  // BEGIN: workaround to prevent duplicated content at end/beginning of page
+  // https://gitlab.pagedmedia.org/tools/pagedjs/issues/126
```

Comment:
@ahus1 Did you try the other alternative (ie. remove newlines when generating the HTML code)?

Reply:
@Mogztter - no, I didn't try fiddling with newlines. This sounded to error prone to me: removing all newlines would have been too much considering pre-formatted content. Removing "just the non-necessary" newlines sounded hard. Not talking about people (like me) using pass-through content to inject some JavaScript here and there...

Reply:
You're probably right, hopefully this workaround won't be needed in the future :wink: